app/assets/javascripts/application.js

```diff
@@ -108,6 +112,7 @@ $(document).ready(function () {
     updateHeader();
 
     $(window).resize(updateHeader);
+    document.addEventListener("turbo:render", updateHeader);
```

Comment:
So obviously this comes down to the whether-to-use-jquery question which was briefly mentioned before, but obviously this is using a raw API for something that we have generally done with jquery in the past.

Reply:
For consistency I dropped jQuery in there.